### DISCUSSION TITLE
Welcome to UV-K5 Modded Firmwares Discussions!

### DISCUSSION THREAD

**Discussion by Tunas1337** [Tunas1337/UV-K5-Modded-Firmwares#19] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jfml2020** [comment]
It seems to me the beginning of a good project... does it really listen in cb27 mhz? Thank you, I joined to follow the project;)

**Reply by Tunas1337** [reply]
@jfml2020 Yes, listening to CB does work, although the sensitivity is not excellent because the filters in the radio are not designed for that task.

**Reply by amnemonic** [reply]
Here https://youtu.be/_0_SXU-ES_M?t=190 guy is listening on CB channel his friend which is located "few" kilometers away. Receiving is quite ok but transmission is not that good, at 7:50 timestamp he did transmission test and it works for about 50 meters distance.

**Reply by jfml2020** [reply]
WoW it seems amazing to me that something so economical and aesthetically the uvk5 radio is not ugly... It is supposed to be the same as other radios but it is seen that it is not like that and it is cool to listen to so many frequencies and modulate  Although it is a bit wrong but it does it .. thanks for the project !!!

**Reply by bg2ke** [reply]
Can you send me the firmware for this K5? Thanks for my email bg2ke11766 @ 163.com

**Reply by jfml2020** [reply]
> ¿Me pueden enviar el firmware para este K5? Gracias por mi correo electrónico bg2ke11766 @ 163.com

https://github.com/Tunas1337/UV-K5-Modded-Firmwares/releases/tag/v0.1

**Reply by bg2ke** [reply]
I've downloaded it. I'm from Jilin province, china. Thank you very much.

**Reply by JEDI-HACKER** [reply]
I've unlocked my radio and tested on 27.555 on recieve is great on FM and AM blown away. However mine does show TX just nothing recieved on my CB radio. Not sure if it the type of firmware I'm using or just my radio doesn't want to transmit. Unfortunately lack of notes for each firmware hack is nothing or very minimal. Lest we forget these are early day's.

I myself am finding a way to custom make the firmware, been a long long while since Baofeng (may they rest in peace) 🤣

**Reply by Tunas1337** [reply]
@JEDI-HACKER For the love of all that is holy, **read the giant warning in the README**. You'll blow up your radio at best, and at worst, cause massive interference on every single multiple of the frequency.
![image](https://github.com/Tunas1337/UV-K5-Modded-Firmwares/assets/12097904/eeca4233-6031-404b-b315-ca82ba916934)

**Reply by JEDI-HACKER** [reply]
Oh thank you. I ran for only a few seconds. This is part of research. I did run it through my 10/11/12 metre antenna.

**Comment by py2pe** [comment]
Hi


Is it possible to use the "Portable Radio CPS" software program after
upgrading with UV-K5-Modded-Firmwares?

Em seg., 12 de jun. de 2023 às 18:36, jfml2020 ***@***.***>
escreveu:

**Reply by jfml2020** [reply]
program is used as the original firmware

**Reply by Tunas1337** [reply]
No one has modified the original software program to work with the expanded frequency ranges.
However, SQ5BPF has made a module that allows Chirp to work with it.

**Reply by speters** [reply]
You know how to build up suspense, don't you?
For those who were also interested (and didn't go through the README of this repo), here's the link: https://github.com/sq5bpf/uvk5-reverse-engineering

**Comment by py2pe** [comment]
Hi Andre

Thanks for your reply.

Could you please send me the link to download ?

Thank you very much.

Regards,

Junior - PY2PE

Em ter., 13 de jun. de 2023 às 05:05, Andrej ***@***.***>
escreveu:

**Comment by jfml2020** [comment]
A question. Does anyone know what the SCREN option (on/off) refers to in the hidden menu?

**Reply by fagci** [reply]
Scrambler on/off.

**Reply by jfml2020** [reply]
But already in the normal menu is to activate or deactivate scramber (SCR on/off)

**Reply by fagci** [reply]
Think, this is same idea, as tx on/off

**Reply by fagci** [reply]
I use FCC standard to show that radio cannot transmit where not needed if police catches me. Same for scrambler =)

**Comment by bg2ke** [comment]
K5 _ 26 _ encrypted _ 18to1300MHz.bin Where can I download this? I really want to experiment with K5

**Reply by Tunas1337** [reply]
It's in the repository itself.
https://github.com/Tunas1337/UV-K5-Modded-Firmwares/blob/main/k5_26_encrypted_18to1300MHz_500k_Step.bin

**Reply by bg2ke** [reply]
I didn't figure out why I couldn't download it. I downloaded the previous one with ease

**Reply by bg2ke** [reply]
I e-mailed you. hoping to get your help. get firmware through you. or the same way I could download it as the last one

**Comment by Andre713** [comment]
Спасибо за вашу работу. Проверял на своей uv-k5. Всё хорошо работает. Есть ли возможность запрограммировать шаг частоты 0,125 0,250 0,500 и 1,000 кгц??

**Reply by Tunas1337** [reply]
Конечно: https://github.com/amnemonic/Quansheng_UV-K5_Firmware/blob/main/uvmod_kitchen/mod_custom_steps.py

**Reply by Andre713** [reply]
Спасибо за ответ, но какой программой открыть этот файл для редактирования?

**Reply by Tunas1337** [reply]
Это Python-скрипта. Там в репозиторий под uvmod-kitchen есть и другие; скачивайте этот репозиторий, инсталлировайте Python, откройте uvmod-kitchen/build.bat, и поставьте "rem" вперед эти строки которые вам не нужно имееть в прошивке.

**Reply by Andre713** [reply]
Спасибо

**Reply by Andre713** [reply]
Добрый день. Не получается ничего.Вы смогли-бы в мод. k5_v2.01.26mod-001.bin прописать шаг  0.025Гц вместо 8.33кГц??? Спасибо.

**Reply by Andre713** [reply]
Нужен для AM модуляции

**Comment by gwe89** [comment]
Would a longer led delay be possible as I hate how quick the screens light goes off

**Reply by Tunas1337** [reply]
Certainly. We found out how to do it by accident and a new uvmod-kitchen patch is underway.

**Comment by F5HCC** [comment]
Hello.
Is it possible to program a frequency with à 1 KHz step on this TRX or is it limited by the embedded synthetizer ?
I would like to use it to monitor SARSAT-COSPAS beacons (406 MHZ band) for my EmComm activities.
Fabrice, F5HCC (France).

**Reply by Tunas1337** [reply]
Hi, the frequency step is changeable using uvmod_kitchen, but the bandwidth of the FM reception is not.
https://github.com/amnemonic/Quansheng_UV-K5_Firmware/blob/main/uvmod_kitchen/mod_custom_steps.py

**Reply by F5HCC** [reply]
Hi.
Thanks a lot for the feedback !!!
Maybe it is possible to get the fm-demodulated signal on a pin of the FM-IF circuit ?
73 from F5HCC.

**Reply by Tunas1337** [reply]
The spec sheet and pinout of the BK4819, which is the radio chip in the UV-K5, is available here:
https://prkele.prk.tky.fi/~ftg/files/datasheets/BK4819_Datasheet_V1.01.pdf

**Comment by jankazimierczak13** [comment]
Hello 
I am just getting into ham radios I was wondering if this mod is also applicable to the Quansheng UV-K5(8) aka UV-K6 ?
If not will you ever release it for the newer model ? 
Cheers

**Reply by Tunas1337** [reply]
Reports say that the firmware can be flashed onto the UV-K6 as well. The radios are identical except for the new design and some changes to the speaker circuitry.

**Reply by schiblinator** [reply]
Is there a way to revert back to the original on the UV-K5(8)? I can't find the firmware. Does it also change to the old menu?

**Reply by Tunas1337** [reply]
amnemonic dumped it a few days ago (the original UV-K5(8) firmware) with the distinct exception that it shows up as 2.01.26 to be able to be flashed on regular UV-K5s. But, yes, the original firmware for the UV-K6 with the different menu is available, you'll just need to modify it *back* again to flash it onto the UV-K6.

**Reply by JEDI-HACKER** [reply]
Oh awesome, would you know what the difference is in menus?

**Reply by Tunas1337** [reply]
Chinese language support, and the split being horizontal instead of vertical.
